### PR TITLE
feat: Add GLN filter param to graphQL

### PR DESCRIPTION
**Pull request by FirestarJes**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -282,7 +282,6 @@ type Mutation {
 
 type Subscription {
   calculationCreated: Calculation!
-  calculationProgress: Calculation!
```
Comment (dzhavat) [inline schema.graphql:285]: question: should this be removed? Doesn't seem related to the other changes.

Comment (FirestarJes) [reply schema.graphql:285]: Ehh, i have no idea where that comes from...

**Review by dzhavat — APPROVED** [review]
